include/triton/core/tritonserver.h

```diff
@@ -879,6 +879,16 @@ TRITONSERVER_DECLSPEC struct TRITONSERVER_Error*
 TRITONSERVER_InferenceTraceRequestId(
     struct TRITONSERVER_InferenceTrace* trace, const char** request_id);
 
+/// Get the child trace, spawned from the parent trace.
+///
+/// \param trace The trace.
+/// \param child_trace Returns the child trace, spawned from the trace.
```

Comment:
Mention caller owns the returned object and must release unless being transferred through other APIs (i.e. `TRITONSERVER_ServerInferAsync`)